common/src/main/java/io/crate/types/GeoPointType.java

```diff
@@ -116,8 +115,8 @@ private static boolean arePointsInRange(double x, double y) {
     }
 
     private static void checkLengthIs2(int actualLength) {
-        Preconditions.checkArgument(actualLength == 2,
-            "The value of a GeoPoint must be a double array with 2 items, not %s", actualLength);
+        assert actualLength == 2
```

Comment:
Isn't this a runtime check method which should validate also without assertions enabled?

Reply:
Ah right,  will change it